README.md

```diff
@@ -10,6 +10,11 @@ Things you may want to cover:
 * System dependencies
 
 * Configuration
+Remember to add to your credentials:
```

Comment:
we also need to add `append :linked_files, 'config/master.key'` in `config/deploy.rb` to use credentials on staging

Reply:
That was one of the main concerns for me yesterday. After some googeling, I end up adding the master key to github > settings > secrets and then refer it into tests.yml You will se that I am using `Rails.application.credentials.api_valid_jwt` in spec/requests/api/v1/users_controller_spec.rb

Reply:
ok, I think I get it now.